### ISSUE TITLE
[Bug 🐞]: https://getmastodon.threefold.io/

### ISSUE THREAD

**Issue by weynandkuijpers** [threefoldtech/www-mastodon#143]
### What happened?

Using an existing mnmonic and ssh key I am not able to use the ssh publi key that I have on my laptop.

Existing ssh public key:
```
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDAOP0h6VImNcxnIBRMoMfbMfb0xwGHDlaPxZ+nu0CL8ATJekVDHDLMGEPdvACfHBe0sqIw/l6jqoEMR4Dzhjgm4bVEUBVEnG1FvkeNB59sT2DOxDCZuqJvjx2M1bJlH8AR/JQXxUQ+zvfTbavc4/zfCuJm4PYNUsmEt/IQmRwLznGOkoJbwYLhKCC3ykZd0EGpmCWgUUYn0ihaaYkyrliQi5Ny00x0s6jOIJg0CG2Xh5xcrkhOfCZMxZAB+/LGQpZ3tu+Cy5jRf8V/JZ8XQmtYM2GmBUZ1KGcMcsGzrtuudn13JeYLtWJBw6A7Q3Fb7dQSCMLC9UA0uMSZk67M6DFV john@RescuedMac
```

I have made sure that this public key does not have any spaces or other unwanted characters in it.

The response is:
![image](https://user-images.githubusercontent.com/13766992/217734588-33f66ac8-53e8-49ee-89f6-58a1d44696fb.png)

It lacks any feedback in terms of why this is not an acceptable public key....

### What did you expect?

Expected the public key to work.  Have been using it in the play.grid.tf and terraform scripts.

### What browsers are you seeing the problem on?

Chrome

**Comment by Mahmoud-Emad** [comment]
Hello @weynandkuijpers 
I tried your ssh key and it saved successfully into my mnemonic, i think you have a space at the endo of your ssh, try to remove it and tell me what happened

**Comment by xmonader** [comment]
let's when fixing this issue make sure to trim the sshkey input before executing anything

**Comment by Mahmoud-Emad** [comment]
@xmonader 
as I mentioned [here](https://github.com/threefoldtech/www-mastodon/issues/144) there is an issue blocking us to run the project locally, maybe it happened because of the new changes of the `ts-client`, @MohamedElmdary about fixing it, once it is fixed I'll try again to fix it.

**Comment by xmonader** [comment]
is it fixed right now @Mahmoud-Emad ?

**Comment by Mahmoud-Emad** [comment]
Should be fixed in #172

**Comment by mohamedamer453** [comment]
Verified on https://getmastodon.threefold.io/

Now the ssh key is trimmed and the spaces before or after it doesn't affect the validation as seen below.

[Get Mastodon (2).webm](https://github.com/threefoldtech/www-mastodon/assets/101194226/1dfaf136-efe7-4c48-8e27-1f2c7b802785)

and to verify that it's being trimmed, i left the ssh key with the spaces that were added in the video above and then i deployed an instance and i was able to ssh to the machine without any issues.

Also verified that the ssh that was originally mentioned in the issue is working fine. and even if a space was added before or after it.

[Get Mastodon (3).webm](https://github.com/threefoldtech/www-mastodon/assets/101194226/1964d3b6-a50a-4da1-8440-92d4dd63c132)
